.github/workflows/update-third-party-notices.yml

```diff
@@ -0,0 +1,26 @@
+name: Check and Update Third Party Notices
+on: workflow_call
+
+jobs:
+  license-check:
+    runs-on: macos-latest
+    steps:
+      - uses: actions/setup-node@v3
+        with:
+          node-version: 18.x
+          registry-url: "https://registry.npmjs.org"
+      - run: npm install -g json@v11
+      - run: npm install -g generate-license-file@2
```

Comment:
I think I like the generate license install here!  Otherwise, I would probably need to have it as a dev dependency in both studio and studio-plugin.  I call `npm install` in both of the subdirectories after removing them from the workspace, so I don't think I could just have it in the top level `package.json`.

Reply:
sure that's fair! can we have a more specific version then, instead of just v2